tests/functional/test_show.py

```diff
@@ -151,6 +152,7 @@ def test_pip_show_is_short(script):
     assert len(lines) <= 10
 
 
+@pytest.mark.network
```

Comment:
yup, sorry, you'll need to add `'--no-index'` to `script.pip('install', ...` to avoid any network call.